packages/wallet/src/services/UtxoTracker.ts

```diff
@@ -36,33 +37,44 @@ export const createUtxoProvider = (
   );
 
 export const createUtxoTracker = (
-  { walletProvider, addresses$, store, transactionsInFlight$, retryBackoffConfig, tipBlockHeight$ }: UtxoTrackerProps,
+  { walletProvider, addresses$, stores, transactionsInFlight$, retryBackoffConfig, tipBlockHeight$ }: UtxoTrackerProps,
   {
     utxoSource$ = new PersistentCollectionTrackerSubject<Cardano.Utxo>(
       () => createUtxoProvider(walletProvider, addresses$, tipBlockHeight$, retryBackoffConfig),
-      store
+      stores.utxo
+    ),
+    unspendableUtxoSource$ = new PersistentCollectionTrackerSubject(
+      (stored) => (stored.length > 0 ? NEVER : concat(of([]), NEVER)),
```

Comment:
I had this in mind:
```suggestion
      () => NEVER
```
But I guess you'd run into the same null value problem?

Reply:
Yes, I'm getting the same errors as I had before when running the tests